src/SynthImage/rusting_augmentation.py

```diff
@@ -0,0 +1,63 @@
+import albumentations as A
+import numpy as np
+from PIL import Image
+
+
+class RustingAugmentation:
+    def __init__(self, original_img_obj):
+        """Initialize the RustingAugmentation class with an image.
+
+        Args:
+            original_img_obj (PIL.Image.Image): The input image to be augmented.
+        """
+        self.original_img_obj = original_img_obj
+
+    def apply_rusting(self):
+        """
+        Apply a rusting effect to the image.
+
+        Returns:
+            PIL.Image.Image: Augmented image with rusting effect.
+        """
+        # Combine various transformations to simulate rust
+        aug = A.Compose(
+            [
+                A.OneOf(
+                    [
+                        A.GaussNoise(
+                            var_limit=(10.0, 50.0), p=0.5
```

Comment:
these numbers needs to be parameterise for better readability as well as gives u options to generate images.